public/externalLibs/streams/stream.js

```diff
@@ -1,289 +0,0 @@
-// stream.js: Supporting streams in the Scheme style, following
```

Comment:
In \public\externalLibs\index.js line 39, the import statement for stream library should be removed.

Reply:
Good catch. Addressed in https://github.com/source-academy/cadet-frontend/pull/781/commits/678bf320e77961e4604de9abde7aabfb5871e31d